### DISCUSSION TITLE
How do we set a font for tiptap?

### DISCUSSION THREAD

**Discussion by synsyst** [awcodes/filament-tiptap-editor#480] [Q&A] [answered]
I'm aware that I can alter that global fonts for the panel - but with tiptap specifically, it's meant to be a way to create content for our site, so how do we enable our users to choose the styling they want?

**Comment by awcodes** [comment, marked as answer]
The plugin doesn't support a font picker tool. You could make a custom extension for it. 

Other option is to scope the font family to the editor via your custom theme to match the site's font family. But you'll all need to use a render hook in filament to load the font's stylesheet.
